src/main/java/org/folio/rest/helper/AwaitingPaymentHelper.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.rest.helper;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.rest.jaxrs.model.AwaitingPayment;
+import org.folio.rest.jaxrs.model.Encumbrance;
+import org.folio.rest.jaxrs.model.Transaction;
+import org.folio.rest.util.MoneyUtils;
+
+import io.vertx.core.Context;
+
+public class AwaitingPaymentHelper extends AbstractHelper {
+
+  public AwaitingPaymentHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
```

Comment:
Awaiting-payment is part of Transactions API and it just a replacement for the PUT /finance/encumbrance/<id>, so I think all this logic can be in Transaction helper.

Reply:
agree